src/app/story/story.component.ts

```diff
@@ -13,16 +14,17 @@ declare var jsPlumb: jsPlumbInstance;
 @Component({
   selector: 'app-story',
   templateUrl: './story.component.html',
-  styleUrls: ['./story.component.css']
+  styleUrls: ['./story.component.css'],
 })
 export class StoryComponent implements OnInit, AfterViewInit, OnDestroy {
 
   loading: boolean;
   story: StoryAPI;
   selectedNode: any;
+  sidebarOpen: boolean;
 
   constructor(private storyService: StoryService, private route: ActivatedRoute,
-              private elRef: ElementRef) {
+              private elRef: ElementRef, private notification: NotificationService) {
```

Comment:
Could you please remove this? It's not used as far as I can tell.

Reply:
I'm having git issues, sorry! For some reason VS code isnt switching when i switch branches on git so this accidentally got pushed